azure-pipelines.yml

```diff
@@ -86,15 +88,25 @@ steps:
     Contents: '*'
     TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
+# - task: Bash@3
+#   displayName: 'Upload to Blob'
+#   condition: and(succeeded(), eq(variables['WHEEL_PUBLISH_FLAG'], 'true'))
+#   inputs:
+#     targetType: 'inline'
+#     script: |
+#       export AZCOPY_SPA_CLIENT_SECRET=$(AZURESDKPARTNERDROPS-SERVICE-PRINCIPAL-KEY)
+#       azcopy login --service-principal --application-id $(APPID)
+#       azcopy copy "$(Build.ArtifactStagingDirectory)/*"  "https://azuresdkpartnerdrops.blob.core.windows.net/drops/azure-iot-edge-tools-iotedgedev/pypi/$(VERSION)/"
+
 - task: Bash@3
```

Comment:
If it's not too much trouble, I would prefer to use 'AzureFileCopy@4' - similar to IoTEdgeHubDev.

Reply:
AzureFileCopy@4 is not supported on the ubuntu vm